.eleventy.js

```diff
@@ -30,6 +30,31 @@ const DEV_MODE = process.env.ELEVENTY_RUN_MODE !== "build" // i.e. serve/watch
 module.exports = function(eleventyConfig) {
     eleventyConfig.setUseGitIgnore(false); // Otherwise docs are ignored
 
+    // Set DEV_MODE_POSTS to true if the context is not 'production'
+    const DEV_MODE_POSTS = process.env.CONTEXT !== "production";
+
+    let hasPrintedMessage = false;
+
+    // Define a filter named 'isFuturePost'
+    eleventyConfig.addFilter('isFuturePost', (date) => {
```

Comment:
If we pass in the post, we can remove the `hasPrintedMessage` altogether and just print: `Include {{ post.title }} scheduled for {{ post.date }}`.

```suggestion
    eleventyConfig.addFilter('isFuturePost', (post) => {
```

Reply:
Thanks! When you say it, it seems so obvious 🤦‍♀️